website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -25,9 +25,26 @@ You must implement the following requirements to create and use cluster peering
 - At least two Kubernetes clusters
 - The installation must be running on Consul on Kubernetes version 0.47.1 or later
 
+### Pre-setup 
+
+1. After provisioning a Kubernetes cluster and setting up your kubeconfig file to manage access to multiple Kubernetes clusters, export the Kubernetes context names for future with `kubectl`. 
+
```

Comment:
```suggestion
### Pre-setup 

1. After provisioning a Kubernetes cluster and setting up your kubeconfig file to manage access to multiple Kubernetes clusters, export the Kubernetes context names for future with `kubectl`. 

```
First, is this a prerequisite or an additional step to the overall workflow? Calling it a pre-step (IMO) is little confusing. To keep it simple, this is either a prerequisite or the first step in the procedure.

Also, "After provisioning a Kubernetes cluster" tracks with the second bullet in the prerequisites section ("At least two Kubernetes clusters"), but "[After]. . . setting up your kubeconfig file to manage access to multiple Kubernetes clusters" is new. I think we should make this step it's own action either as part of the prerequisites or as the first (second?) step in the overall procedure.

Does that make sense?

Reply:
It is not an explicit pre-req but makes the overall process of issuing commands a lot easier if you are working with multiple clusters.